wwwroot/inc/snmp.php

```diff
@@ -4436,6 +4454,30 @@
 		'text' => ' J9049A: 24 RJ-45/10-100-1000T',
 		'processors' => array ('procurve-21-to-24-combo-1000SFP', 'procurve-chassis-1000T'),
 	),
+	'14988.2' => array
+	(
+    	'dict_key' => 3651,
+     	'text' => 'Mikrotik  CSS326-24G-2S+RM 24 port Gigabit Ethernet switch with two SFP+ ports',
+     	'processors' => array ('Mikrotik-25-to-26-SFP+','Mikrotik-GigabitEthernet-1000TX'),
+	),
+	'9.1.574' => array
```

Comment:
please sort by OID.  it would be helpful to sort the rest of the entries while you're at it.  there are only a few stragglers